apps/site/src/_pages/docs/5_faq.mdx

```diff
@@ -134,9 +134,15 @@ Because those inserting blocks are often non-technical end users, this isn't alw
   - which blocks users are able to discover and insert within your environment
   - which blocks a user inserts are ultimately able to be run in either a sandboxed or unsandboxed fashion
 
-Today you might choose to allow/deny blocks on the basis of their authorship (published namespace), or based on their _verification status_. In the future we also plan to expose additional Þ _Hub_ metadata which may prove useful in the creating and maintaining whitelists.
+Today you might choose to allow/deny blocks on the basis of their authorship (published namespace), or based on their _verification status_.
+  
+In the future we also plan to expose:
+- additional Þ _Hub_ metadata which may prove useful in the creating and maintaining whitelists;
+- permission grants for blocks and block types, that allow for users to control the level and types of access blocks have (e.g. the ability to access information created through the Graph Service outside of themselves) or access to other services (e.g. making network requests).
```

Comment:
This would involve a more complicated sandboxing solution we'll need to think about further.
```suggestion
- permission grants for blocks and block types, that allow for users to control the level and types of access blocks have (e.g. the ability to access information created through the Graph Service outside of themselves) or access to other services.
```